README.md

```diff
@@ -1 +1,63 @@
-# todolist-app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My ToDo List App
+
+> Description of Awesome Books.
+This is a todo list I built to store and list tasks with just Html Css and Javascript making use of webpack. <br>
+No frameworks Used
+
+## Description
+This app is created to manage tasks, Create and delete
+
+
+
+## Built With
+
+Html and Css and Js 
+
+## Library 
+[WebPack](https://webpack.js.org)
+
+
+## Getting Started
+
+git clone https://github.com/mudasiri/todolist-app.git
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+You have basic knowledge of Html, Css, and code editor.
+### Setup
+
+### Install
+npm install <br>
+### Usage
+Free feel to use the project as you like
+
+
+### Run tests
+You can use vscode extension or just run the index file. (I have yet to set up local server)
+
+
+
+### Deployment
+
+## Contact me
+
+<a href="https://github.com/mudasiri/">Github</a>
+
+<a href="https://twitter.com/mudasiri">Twitter</a>
+
+<a href="https://www.linkedin.com/in/mudasiri/">LinkedIn</a>
+
+## Issues
+You can make a PR or make an issue if you found problems
+
+## Show your support
+Give me a star if it is your taste
+
+
+## Acknowledgments
+
+Great thanks to microverse
```

Comment:
- [ ] OPTIONAL
It's recommended that you add an MIT license, or any other similar license to your ReadMe for this project.